### ISSUE TITLE
T-range for dipole not tunable

### ISSUE THREAD

**Issue by thegarzo** [Non-Equilibrium-QCD/McDipper#6]
I found that when changing the Thickness range and number of points, there seems to be some inconsistencies. If I change T in params_ipsat.h, then the code does not run. 
We need to fix this

**Comment by zhujie12138** [comment]
I think the logic is 
- We first check table of charges. If found, we don't care about dipole table at all. That's why the code doesn't run in your case.
- If the table of charges was not found, we will check dipole table then. 

So, if you think it is necessary to check both tables, I can implement this. 
But considering generation of dipole table is very fast, people can easily set enough range or precision at very begginning.

**Comment by thegarzo** [comment]
You are right, the issue was that I used an old version of a config and forgot to write in the new section of the Thickness... However, it may be good to keep this issue to revise whether we can simplify this later
